src/Accordion/AccordionHeader.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import StyledAccordionHeader from './StyledAccordionHeader';
+import StyledAccordionHeading from './StyledAccordionHeading';
+import StyledAccordionHeadingButton from './StyledAccordionHeadingButton';
+import Text from '../Text';
+
+const AccordionNoteHeader = ({ note }) =>
+  note && (
+    <Text fontSize="size4" textAlign="right" color="grey80">
```

Comment:
is `grey80` still a thing after https://www.pivotaltracker.com/story/show/163805431?

Reply:
Awesome catch, thank you.